src/mgr/PyModuleRunner.cc

```diff
@@ -99,6 +99,16 @@ void PyModuleRunner::log(int level, const std::string &record)
 #define dout_prefix *_dout << "mgr " << __func__ << " "
 }
 
+void PyModuleRunner::cluster_log(const std::string &channel, clog_type prio,
```

Comment:
If you move this function (and the audit_clog member) into ActivePyModule, then you can avoid the modification the StandbyMgrModule instantiations.  I think it's clearer if this function is only accessible where it's meant to be used (active modules) rather than in both standby and active.

Reply:
Done